skbio/stats/power.py

```diff
@@ -209,6 +219,10 @@ def subsample_power(test, samples, draw_mode='ind', alpha_pwr=0.05,
     ValueError
         If the `counts_interval` is greater than the difference between the
         sample start and the max value, the function raises a ValueError.
+    ValueError
+        There are not an equal number of groups in `samples` and in `ratios`.
+    TypeError
+        `test` does not return a float or a 1-dimensional numpy array.
 
 
     Examples
```

Comment:
Do you think it would be worth it to put another example with the new ratio parameter?